prisma/schema.prisma

```diff
@@ -41,16 +41,17 @@ model Session {
 }
```

Comment:
for some reason, when i access NextAuth session, I can't access the userId. This is causing authentication problems (for edit and delete funcs, for example)

Reply:
Found out that I had to edit the NextAuth config file's "session" callback to return our user's id with other session info.